.env

```diff
@@ -1,13 +1,14 @@
 REACT_APP_APPNAME='Axial'
-REACT_APP_NODE_ADDRESS='http://localhost:8545'
-#'https://node.snowapi.net'
+REACT_APP_NODE_ADDRESS='https://node.snowapi.net'
+#'http://localhost:8545'
```

Comment:
should we remove this unnecessary code?

Reply:
no, it's to test local and main endpoint

Reply:
Maybe we can have both and check node env to choose the correct one?